### PR TITLE
Update rubygem-kubeclient and fog-kubevirt

### PR DESCRIPTION
**Pull request by dgoetz**
This is required for relaxing the dependency on http gem and to fix a vulnerability in kubeclient.

### CODE REVIEW

File: packages/plugins/rubygem-kubeclient/rubygem-kubeclient.spec [hunk 1]
```diff
@@ -1,84 +1,56 @@
...
         %{buildroot}%{gem_dir}/
 
 %files
 %dir %{gem_instdir}
+%exclude %{gem_instdir}/.github
 %exclude %{gem_instdir}/.gitignore
 %exclude %{gem_instdir}/.rubocop.yml
-%exclude %{gem_instdir}/.travis.yml
 %license %{gem_instdir}/LICENSE.txt
+%doc %{gem_instdir}/RELEASING.md
```
Comment (ekohl) [inline packages/plugins/rubygem-kubeclient/rubygem-kubeclient.spec:53]: I think we can exclude this too.

File: packages/plugins/rubygem-recursive-open-struct/rubygem-recursive-open-struct.spec [hunk 2]
```diff
@@ -1,70 +1,53 @@
...
 
 %description
 RecursiveOpenStruct is a subclass of OpenStruct. It differs from
 OpenStruct in that it allows nested hashes to be treated in a recursive
-fashion.
+fashion. For example:
+ros = RecursiveOpenStruct.new({ :a => { :b => 'c' } })
+ros.a.b # 'c'
+Also, nested hashes can still be accessed as hashes:
+ros.a_as_a_hash # { :b => 'c' }.
```
Comment (ekohl) [inline packages/plugins/rubygem-recursive-open-struct/rubygem-recursive-open-struct.spec:26]: I'd drop the example
```suggestion
fashion.
```

File: packages/plugins/rubygem-recursive-open-struct/rubygem-recursive-open-struct.spec [hunk 3]
```diff
@@ -78,17 +61,21 @@ cp -pa .%{gem_dir}/* \
 
 %files doc
 %doc %{gem_docdir}
-%exclude %{gem_instdir}/.document
+%doc %{gem_instdir}/.document
```
Comment (ekohl) [inline packages/plugins/rubygem-recursive-open-struct/rubygem-recursive-open-struct.spec:64]: I think we can keep excluding this
```suggestion
%exclude %{gem_instdir}/.document
```

**Review by ekohl — COMMENTED** [review]
Thanks! Some minor comments.

**Comment by dgoetz** [comment]
Fixed your comments, thanks. Now waiting for it to get green.

**Review by ekohl — APPROVED** [review]
I forgot about this, but looks good.

**Comment by ekohl** [comment]
```console
$ koji -p katello add-pkg --owner kojiadm foreman-plugins-nightly-el8 rubygem-jsonpath
Adding 1 packages to tag foreman-plugins-nightly-el8
```

**Pull request merged**